docs/pages/guides/customizing-metro.md

```diff
@@ -26,6 +28,104 @@ You can find a complete list of supported options in the Metro docs: [Configurin
+| Config file      | `metro.config.js`    | `webpack.config.js`    |
+| Default config   | `@expo/metro-config` | `@expo/webpack-config` |
+| Fast Refresh     | WIP (coming soon)    | ❌                     |
+| Tree Shaking     | ❌                   | ✅                     |
+| CSS Handling     | ❌                   | ✅                     |
+| Asset Manifests  | ❌                   | ✅                     |
+| Bundle Splitting | WIP (pending native) | ✅                     |
+
+Note that aliases, resolution, and other bundler features are now universal across platforms as well!
+
+### Web Support: How?
+
+To enable Metro web support, be sure to use Expo SDK 46, then modify your Expo config (`app.json`, or `app.config.js`) to enable the feature using the `expo.web.bundler` field:
+
+```json
+{
+  "expo": {
+    "web": {
+      "bundler": "metro"
+    }
+  }
+}
+```
+
+#### Web Support: Development
+
+To start the development server:
+
+<Terminal cmd={["$ npx expo start --web"]} />
+
+Or start normally and press `w` in the Terminal UI.
+
+#### Web Support: Production
+
+> TL;DR: Use `npx expo export` instead of `npx expo export:web` or `expo build:web`.
+
+You can bundle the project for hosting just like you would for native:
+
+<Terminal cmd={["$ npx expo export"]} />
+
+You should see something like this:
+
+```
+app $ npx expo export
+Starting Metro Bundler
+iOS ./index.tsx ░░░░░░░░░░░░░░░░  4.0% (  8/132)
+Android ./index.tsx ░░░░░░░░░░░░░░░░  0.5% (  3/129)
+Web ./index.tsx ░░░░░░░░░░░░░░░░  4.0% ( 5/5
+```
+
+You can skip bundling for all platforms by using the `--platform` flag:
+
+<Terminal cmd={["$ npx expo export --platform web"]} />
+
+Your output will be found in the `dist` folder.
+
+The output can be tested locally by using the [Serve CLI](https://www.npmjs.com/package/serve).
+
+<Terminal cmd={["$ npx serve dist"]} />
+
+You can deploy the website using any popular web host, follow any of the guides in [publishing websites](/distribution/publishing-websites/), just substitute the `web-build/` folder for the `dist/` folder.
+
+#### Static Files
+
+Expo's Metro implementation supports hosting static files from the dev server by putting them in the root `public/` folder, this is akin to many other web frameworks. In Expo Webpack we default to using the `web/` folder.
+
+When exporting with `npx expo export` we copy the contents of the `public/` folder into the `dist/` folder, meaning your app can expect to fetch these assets relative to the host URL.
+
+The most common example of this is the `public/favicon.ico` which is used by websites to render the tab icon.
+
+You can overwrite the default `index.html` in Metro web by creating a `public/index.html` file in your project.
+
+In the future, this will work universally across platforms with EAS Update hosting. Currently the feature is web-only based on the static host used for the native app, e.g. the legacy Expo service updates does not support this feature.
```

Comment:
```suggestion
In the future, this will work universally across platforms with EAS Update hosting. Currently the feature is web-only based on the static host used for the native app, for example, the legacy Expo service updates does not support this feature.
```

Let's use the full form instead of Latin abbreviation here 👍